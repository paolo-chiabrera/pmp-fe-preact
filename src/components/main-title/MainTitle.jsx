import { h } from 'preact';

export default function MainTitle() {
  return (
    <div class="main-title-wrapper">
      <div class="main-title">
        <span>P</span>ick<span>M</span>e<span>P</span>lease
      </div>
      <div class="sub-title">"Lesser artists borrow, great artists steal"</div>
    </div>
  );
}
