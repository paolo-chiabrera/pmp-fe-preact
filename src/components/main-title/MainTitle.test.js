import { h } from 'preact';

import render from 'preact-render-to-string';

import MainTitle from './MainTitle.jsx';

describe('Given a MainTitle component', () => {
  it('should render correctly', () => {
    const tree = render(<MainTitle />);

    expect(tree).toMatchSnapshot();
  });
});
