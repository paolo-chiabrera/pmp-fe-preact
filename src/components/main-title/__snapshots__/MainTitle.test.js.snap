// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a MainTitle component should render correctly 1`] = `"<div class=\\"main-title-wrapper\\"><div class=\\"main-title\\"><span>P</span>ick<span>M</span>e<span>P</span>lease</div><div class=\\"sub-title\\">&quot;Lesser artists borrow, great artists steal&quot;</div></div>"`;
